.devcontainer/Dockerfile

```diff
@@ -17,33 +16,22 @@ ARG USERNAME=node
 # (See https://github.com/microsoft/vscode-dev-containers/tree/master/containers/javascript-node/.devcontainer)
 FROM mcr.microsoft.com/vscode/devcontainers/javascript-node:${NODE_VERSION}
 
```

Comment:
The diff below is messy, but the changes are very minor:
* I moved installation of docker CLI earlier so it would be cached while I fiddled with stuff below.
* I changed DEBIAN_FRONTEND from an ENV to an inline export so it would persist
* I removed setting term (already xterm-256color)
* I used a separate RUN for clean